aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -89,6 +89,16 @@ class DispatchTable final {
   , dispatchKeyExtractor_(DispatchKeyExtractor::make(schema))
   , operatorName_(toString(schema.operator_name())) {}
 
+  // a dispatch table may be default constructed with only an
+  // operator name.  Such a dispatch table is not callable until
+  // the schema is provided
+  DispatchTable(const OperatorName& op_name)
+  : kernels_()
+  , catchallKernel_()
+  , dispatchKeyExtractor_(DispatchKeyExtractor::makeUninitialized())
+  // TODO: this toString call is a travesty
```

Comment:
I don't know! It's what the code did before.

Reply:
you can also store it as `OperatorName` if you want to avoid the `toString`. It's only used for error messages in here, removing the toString here would move it towards where the error message is created.

Reply:
OK, I did this change in https://github.com/pytorch/pytorch/pull/34620